### ISSUE TITLE
Piscope shows pin activity but RFSniffer does not report it

### ISSUE THREAD

**Issue by bitstrike** [timleland/rfoutlet#6]
Hi - I'm not able to get RFSniffer to report any signal on my RPi 3 and wondering if I have a software issue (running RaspiOS bullseye). I ran Piscope to check GPIO27 and there certainly seems to be signal on that pin.

In this [screenshot](https://imgur.com/0kOm0lvl.png), I pressed a button on my 433mhz [BN-LNK remote](https://imgur.com/5QhLcBVl.png) and could certainly see some signal coming over it (in red box). The signal would repeat with each button press. 

Does anything look amiss or is the pin just too noisy for RFSniffer to pick up the signal from the remote? My Pi is wired like [this](https://imgur.com/HzTElQgl.png). I'm using WiringPi from [here](https://github.com/WiringPi/WiringPi.git)

**Comment by timleland** [comment]
Hi,

This project is pretty old and I have not run it in years. Are you seeing any error messages?

**Comment by bitstrike** [comment]
No, no error messages. It just doesn't produce any output and I'm trying to track it down. I had posted the question on a forum and someone suggested submitting a bug report but I don't know for sure this is actually a bug.
